src/components/JsonWebMap.vue

```diff
@@ -290,10 +296,44 @@ function showDocumentation(id: string) {
   }
 }
 
+function getGenusTreeCountLabel(sel: SpeciesItem) {
+  return new Intl.NumberFormat().format(sel['GENUS TREE COUNT'])
+}
+
+function getGenusShareLabel(sel: SpeciesItem) {
+  return new Intl.NumberFormat().format(Number.parseFloat(sel['GENUS SHARE'].replace('%', ''))) + '%'
```

Comment:
Just for i18n purposes, I'd create a custom numberFormat wrapper to handle the language (en-US... fr-FR) or other ?

Because Right now it's using  commas for the thousand separators. and in switzerland I think it's more of a spaces for thousands and comma for the decimals which could be confusing..

I'm not sure we should confirm with the researcher

Reply:
Correct, thanks for the reminder but I plan to do it when I will integrate i18n.